cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -673,6 +673,25 @@ func getEnvWithDefaultInt64(k string, defaultVal int64) int64 {
 	return i
 }
 
+func initializeCompoundShardCounter(indexDir string) {
+	d, err := os.Open(indexDir)
+	if err != nil {
+		log.Printf("initializeCompoundShardCounter: %s\n", err)
+		return
+	}
+	defer d.Close()
+	fns, _ := d.Readdirnames(-1)
```

Comment:
nit: What does `fns` stand for here?

Reply:
I think we could use https://pkg.go.dev/io/fs#Glob here.

Reply:
`fns` stands for filenames, we use it in cleanup.go, too.

Reply:
Glob sounds good. I chose filepath.Glob over fs.Glob because we already use it in watcher.go and other places. Any reason to prefer fs.Glob here?